Misc/NEWS.d/next/Library/2025-04-14-23-00-00.gh-issue-132527.kTi8T7.rst

```diff
@@ -0,0 +1,2 @@
+Added the missing 'w' typecode to the error message which is issued by
+array.array() if used with an unknown typecode like array('x').
```

Comment:
The news entry could be a bit more concise. How about this?
```suggestion
Improve the error message when an invalid typecode is passed to :class:`array.array`.
```

Reply:
See my alternative below.